cms/static/js/spec/video/transcripts/editor_spec.js

```diff
@@ -31,10 +31,21 @@ function($, Backbone, _, Utils, Editor, MetadataView, MetadataModel, MetadataCol
                 type: MetadataModel.GENERIC_TYPE,
                 value: 'display value'
             },
-            models = [DisplayNameEntry, VideoListEntry],
+            VideoIDEntry = {
+                default_value: 'test default value',
+                display_name: 'Video ID',
+                explicitly_set: true,
+                field_name: 'edx_video_id',
```

Comment:
Looks like these fields are being used in models to fake the backend data, so let's leave them as it.